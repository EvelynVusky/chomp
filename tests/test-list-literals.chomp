int main() {
    println([1, 2, 3]);
    println(car [1, 2, 3]);
    println(cdr [1, 2, 3]);
    println(1 :: [2, 3]);
    list string s = ["hello", "hi", "chomp"];
    println(["hello", "hi", "chomp"]);
    println(s);
    println(car s);
    println(cdr s);
    println("no" :: s);
    println(['a','b','c']);
    println([{{0}}, {{1}}]);
    return 0;
}