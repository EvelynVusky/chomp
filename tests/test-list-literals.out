[ 1 2 3 ]
1
[ 2 3 ]
[ 1 2 3 ]
[ hello hi chomp ]
[ hello hi chomp ]
hello
[ hi chomp ]
[ no hello hi chomp ]
[ a b c ]
[ 0 1 ]
